### ISSUE TITLE
IOS XE Always On Sandbox: Ansible playbook can't run on IOS XE Always-on

### ISSUE THREAD

**Issue by annegentle** [CiscoDevNet/learning-labs-issues#121]
Cannot get the playbook to run on this step: https://developer.cisco.com/learning/modules/intro-ansible-iosxe/ansible-ios-modules/step/2 

On VPN:
socket_path does not exist or cannot be found.

Off VPN:
```
TASK [GATHERING FACTS] *********************************************************
fatal: [ios-xe-mgmt.cisco.com]: FAILED! => {"msg": "Authentication timeout."}
```

**Comment by smupad** [comment]
@annegentle @klevenstein - Is it possible to get more details on this issue?  The learning lab (https://developer.cisco.com/learning/modules/intro-ansible-iosxe/ansible-ios-modules/step/2) uses IOS XE **Always On** sandbox. User does not have to VPN to access it. So the "On VPN/Off VPN"message is not clear.

**Comment by annegentle** [comment]
I want to say I was on or off Cisco VPN, not a reservation VPN. Let me try again and put in more info.

**Comment by annegentle** [comment]
Seems safe to close as we can't necessarily account for all networks latency settings.
